monai/metrics/conformal_risk.py

```
@@ -0,0 +1,477 @@
+            should check the achieved risk with :class:`Coverage` / :class:`SetSize`.
+        """
+        if not self._scores:
+            raise RuntimeError("No calibration data accumulated; call accumulate(probs, labels) first.")
+        n = len(self._scores)
+        device, dtype = self._scores[0].device, self._scores[0].dtype
+        lam_grid = self.lam_grid.to(device)
+        n_lam = lam_grid.numel()
+        # Sum each image's per-lambda loss; images vary in size so we loop per image but
+        # vectorize over the whole lambda grid (n_lam acts as the batch dim into loss_fn).
+        risk_sum = torch.zeros(n_lam, device=device, dtype=torch.float32)
+        for scores_i, labels_i in zip(self._scores, self._labels):
```

Ruff (0.15.17)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)